examples/agent-pool-terraform/token.tf

```diff
@@ -0,0 +1,21 @@
+resource "random_string" "token" {
+  count  = var.agent_count
+  length = 32
+}
+
+resource "teleport_provision_token" "agent" {
+  count = var.agent_count
+  spec = {
+    roles = [
+      "App",
+      "Db",
+      "Discovery",
+      "Kube",
+      "Node",
+    ]
```

Comment:
This is not directly related to the change, but it seems you're adding new roles. Could you make the roles a variable and tell the user to set only the needed ones? Granting all roles by default is a security anti-pattern.